### ISSUE TITLE
Spawnmob certain mobs do not spawn.

### ISSUE THREAD

**Issue by Marcus101RR** [Pryaxis/TShock#1350]
Source: Pre-Release 2 of TShock Mintaka 4.3.21 for Terraria 1.3.4.4 (APIv2.0, OTAPI v2.0.0.15)

Certain mobs such as (Eater of Worlds, Worms Types) do not seem to spawn. I will be testing more to find any other mob types.

[Image]
![image](https://cloud.githubusercontent.com/assets/8344293/21275866/b9d07846-c39c-11e6-8f30-1feaf0c5ce17.png)

**Comment by ProfessorXZ** [comment]
Just to keep track of the issue: 

```

enerdy [11:46 PM]  
So not exactly our problem, most likely could be added as a feature

[11:46]  
Can add the [PR Wanted] tag

marcus101rr [11:56 PM]  
It worked before

[11:56]  
why did it stop now?

[11:57]  
Also, it doesn't work on SB

professorx [11:57 PM]  
@enerdy Maybe, but don't jump to conclusions

[11:57]  
Worm AI types caused issues in the earlier releases

enerdy [11:57 PM]  
I really, really doubt its a tshock issue

[11:57]  
Or rather in this case its more of a debate about the scope of the command

professorx [11:57 PM]  
The last time I checked the code TSAPI did weird stuff

marcus101rr [11:57 PM]  
It worked on old tshock all the time.

professorx [11:57 PM]  
Like, if there are no players in the corruption zone it just despawned

[11:58]  
But apparently vanilla doesn't do that
````

**Comment by Kojirremer** [comment]
Skeletron does not spawn either. Prime does without issue.

**Comment by tlworks** [comment]
EoW, Moon Lord, Cultist, Big Mimics, Dune Splicers, Wyvern, Phantasm Dragon, Some new monsters get transformed into older ones.

gl

**Comment by ProfessorXZ** [comment]
Was resolved in [9050543](https://github.com/Pryaxis/TShock/commit/90505435bafefac46b50dee49513fc48d96530de). Closing
